pom.xml

```diff
@@ -218,6 +218,22 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>${maven-javadoc-plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <id>generate-javadocs</id>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>javadoc</goal>
+                            </goals>
+                            <configuration>
+                                <detectJavaApiLink>false</detectJavaApiLink>
+                                <source>8</source>
+                                <notree>true</notree>
+                                <includeDependencySources>false</includeDependencySources>
+                                <doclint>none</doclint>
+                            </configuration>
+                        </execution>
+                    </executions>
                 </plugin>
```

Comment:
This will likely increase our build time because we are generating Javadocs now as part of the build.

Reply:
Build took at least 15 mins longer after the change. We probably should not build it then or at least disable it for service modules

Reply:
- Are we going to use these Java Docs in https://github.com/aws/aws-sdk-java-v2/blob/master/buildspecs/release-javadoc.yml ?
- Why didn't we consider adding it as parallel  step , since we have failures due to intermittent test failures.

Reply:
Nope, they are part of the build process. I tested it and it did not increase the build time w/ service modules disabled.

I considered creating another codebuild job to run the same command as release-javadoc, but the whole job can take more than one hour and it works with the largest instance, so I went with this approach.